tests/Unit/NumericalAlgorithms/DiscontinuousGalerkin/Tags/Test_Formulation.cpp

```diff
@@ -0,0 +1,22 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <string>
+
+#include "Framework/TestCreation.hpp"
+#include "Helpers/DataStructures/DataBox/TestHelpers.hpp"
+#include "NumericalAlgorithms/DiscontinuousGalerkin/Formulation.hpp"
+#include "NumericalAlgorithms/DiscontinuousGalerkin/Tags/Formulation.hpp"
+
+SPECTRE_TEST_CASE("Unit.DiscontinuousGalerkin.Tags.Formulation",
+                  "[Unit][Evolution]") {
+  TestHelpers::db::test_simple_tag<dg::Tags::Formulation>("Formulation");
+  CHECK(
+      TestHelpers::test_creation<dg::Formulation, dg::OptionTags::Formulation>(
+          "Strong") == dg::Formulation::StrongInertial);
```

Comment:
I would strongly prefer for the creation string to match the actual name of the enum -- having these differ is just inviting confusion down the line. I don't care if you rename the string or the enum :)